query/common.d.ts

```diff
@@ -269,6 +277,39 @@ export interface Bindings extends Iterable<[RDF.Variable, RDF.Term]> {
   ) => Bindings;
 }
 
+export interface Paths extends Iterable<Bindings> {
+  type: 'paths';
+  /**
+   * The number of variable-value pairs.
+   */
+  size: number;
```

Comment:
I think this should rather be described as the length of the path. For a given query each step in the path has the same number of variable-value pairs, although it may differ for different queries.

Reply:
And although `size` is used for `Bindings`, for a path I think `length` makes more sense as a name.